nifi-bootstrap/pom.xml

```diff
@@ -34,13 +34,26 @@ language governing permissions and limitations under the License. -->
             <artifactId>nifi-utils</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
```

Comment:
by pulling this dependency in here you'll also need to handle the dependency properly in the rpm assembly.  Otherwise when the RPM is built it wont work.  True for any transitive deps of this as well.